src/modules/clearCompleted.test.js

```diff
@@ -1,34 +1,46 @@
-import clearCompleted from "./clearCompleted";
-import save from "./filtered";
+import clearCompleted from './clearCompleted.js';
+import save from './filtered.js';
+
+describe('clearCompleted', () => {
+  test('should remove completed tasks', () => {
+    const tasks = [
+      { description: 'Task 1', completed: true },
+      { description: 'Task 2', completed: false },
+      { description: 'Task 3', completed: true },
+    ];
+    const expected = [{ description: 'Task 2', completed: false }];
+    expect(clearCompleted(tasks)).toEqual(expected);
+  });
+});
```

Comment:
- [ ] Ouch! this is really close but it seems like one of the tests is failing. 👇
  ![image](https://github.com/ProgramKingAbel/Todo-List/assets/104968236/76b082a6-4256-4ac0-a734-9280b2418c93)

Reply:
We have removed code that was deterring functionality. Thank you for pointing that out. 🥂 🍾